frontend/app/components/marketplace/productos-pagination.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Pagination,
+  PaginationContent,
+  PaginationItem,
+  PaginationPrevious,
+  PaginationLink,
+  PaginationEllipsis,
+  PaginationNext,
+} from "../ui/pagination";
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "../ui/select";
+
+export const ProductsPagination = () => {
+  return (
+    <section className="w-full pb-10 mt-2 flex flex-col justify-center items-start gap-5 
+    md:flex-row md:items-center md:gap-0 md:max-w-2xl lg:max-w-3xl xl:max-w-4xl 2xl:max-w-5xl">
+      <div className="w-full gap-2 flex justify-center items-center md:w-[50%] md:justify-start">
+        <label className="opacity-80">Show result:</label>
+        <Select>
+          <SelectTrigger className="w-[180px]">
+            <SelectValue placeholder="10" />
+          </SelectTrigger>
+          <SelectContent>
```

Comment:
Add min-w-[70px] here

Reply:
Quiere que lo agregue al componente "SelectTrigger"?

Reply:
In the `SelectContent`